src/plugins/index_pattern_management/public/components/edit_index_pattern/tabs/tabs.tsx

```diff
@@ -183,7 +183,7 @@ export function Tabs({ indexPattern, saveIndexPattern, fields, history, location
           return (
             <>
               {useUpdatedUX && <EuiSpacer size="m" />}
-              <Wrapper>
+              <Wrapper paddingSize="m">
```

Comment:
Do we need to take useUpdatedUX into consideration? Here we will have 2x paddingM vertically if useUpdatedUX is true.

Reply:
Sorry I'm not quite clear on how useUpdatedUX leads to 2x paddingM, but I see it work as expected when useUpdatedUX is true
<img width="1702" alt="Screenshot 2024-11-04 at 10 35 37" src="https://github.com/user-attachments/assets/ad3428d3-c913-418b-88ff-5071b3bbd6bf">